fluss-client/src/test/java/com/alibaba/fluss/client/table/scanner/log/RemoteLogDownloaderTest.java

```diff
@@ -163,4 +172,19 @@ private static List<RemoteLogSegment> buildRemoteLogSegmentList(
         }
         return remoteLogSegmentList;
     }
+
+    private static Long remoteLogSegmentFilesLength(
+            List<RemoteLogSegment> remoteLogSegments, FsPath remoteLogTabletDir, int segmentNum) {
+        return remoteLogSegments.stream()
+                .limit(segmentNum)
+                .mapToLong(
+                        segment ->
```

Comment:
We can get the bytes length via `RemoteLogSegment#segmentSizeInBytes`.

Reply:
@wuchong, `RemoteLogSegment#segmentSizeInBytes` is set with `Integer.MAX_VALUE`. Therefore, we could get the length of the log segment file.